src/lightning/fabric/strategies/launchers/subprocess_script.py

```diff
@@ -143,6 +143,7 @@ def _basic_subprocess_cmd() -> Sequence[str]:
 
 def _hydra_subprocess_cmd(local_rank: int) -> Tuple[Sequence[str], str]:
     import __main__  # local import to avoid https://github.com/Lightning-AI/lightning/issues/15218
+    from hydra.core.hydra_config import HydraConfig
```

Comment:
The minimun supported version according to our imports is `hydra-core>=1.0.7`, would you mind double checking that this import works for that version. Thx

Reply:
nvm we import it in other places too